.circleci/config.yml

```diff
@@ -195,13 +195,12 @@ jobs:
             wget https://julialang-s3.julialang.org/bin/linux/x64/1.0/julia-1.0.3-linux-x86_64.tar.gz
             tar xfz julia-*-linux-x86_64.tar.gz
             ln -s $(pwd)/julia-*/bin/julia /usr/bin/julia
-            conda create -y -n build conda-build conda-verify
-            conda activate build
-            # Log versions
-            dpkg -l > /logs/versions.txt
-            conda env export > /logs/build_environment.yml
+            conda update -y -n base conda
+            conda install -y python=3.9.2
+            conda install -y -c conda-forge conda-build conda-verify
```

Comment:
Why do you want to use the base environment and upgrade that to a very specific python version rather than a fresh environment with all the latest packages? The latter seems more flexible as it will automatically pick up the latest version of the `conda-build` and `conda-verify` packages, so will require less maintenance.

Reply:
Hi @valeriupredoi did you have a chance to look into this already?

Reply:
sorry @nielsdrost I commented but it looks like I didn't submit the review comment, gitHub is being fussy :grin: